### ISSUE TITLE
In two syllable word: the second neutral syllable can get tone color later on

### ISSUE THREAD

**Issue by mawis** [luoliyan/chinese-support-redux#80]
**Describe the bug**
A two syllable word where the second syllable is in the neutral tone is first colored correctly, but when I pass through the 拼音 field, the second syllable gets the color of the first syllable.

**To Reproduce**
1. Create a new card
2. Fill in 爽快 as the 汉字
3. Press the tab key one time
4. Press the tab key several more times until you reach below the 拼音 field.

**Expected behavior**
First syllable should be colored as 3rd tone, second syllable should be colored in the neutral tone.

**Screenshots**
After step 3 everything seems to be correct:
![20190803_recoloring_before](https://user-images.githubusercontent.com/2042030/62410019-364f9e00-b5e0-11e9-8a17-36b509ae4e84.png)

But after step 4 the coloring is incorrect:
![20190803_recoloring_after](https://user-images.githubusercontent.com/2042030/62410021-423b6000-b5e0-11e9-8c56-fc8d6b3d2e2a.png)

**Specs (please complete the following information):**
 - OS: Linux (NixOS 19.03)
 - Anki Version 2.1.9
- Chinese Support Version v0100

**Comment by mawis** [comment]
I just noticed: also the tone mark in the Zhuyin/Bopomofo is set incorrectly after the second syllable. It seems the separation into two syllables doesn't work correctly.

**Comment by mawis** [comment]
Some “recoloring” after passing the 拼音 field just happened with 流浪。

**Comment by luoliyan** [comment]
Would you be so kind as to test against the current version on AnkiWeb. Hopefully I've caught most of the issues.

**Comment by mawis** [comment]
I have Anki 2.1.14 Standard on Windows 10, 64bit (i.e. different computer than the one from which I reported the problem). I updated the plugin to v0111. Still the same thing is happening.

**Comment by luoliyan** [comment]
![image](https://user-images.githubusercontent.com/24988028/63032211-8a1d8980-bef4-11e9-92bf-2b87434823d9.png)

**Comment by luoliyan** [comment]
I tabbed through all the fields and can't replicate this now. As I mentioned in #79, it probably has to do with field names. I'll close this for now, but feel free to reopen if I'm wrong.
